### DISCUSSION TITLE
Automatic power mode switching

### DISCUSSION THREAD

**Discussion by shuiying-yue** [BartoszCichecki/LenovoLegionToolkit#1521] [Q&A] [answered]
After the computer reboots, the power mode is automatically changed to “Balanced Mode” instead of remaining in the original “Custom Mode”.

**Comment by BartoszCichecki** [comment, marked as answer]
Yes that is normal on some laptops. You can always configure an Action to switch to custom on startup.

**Comment by shuiying-yue** [comment]
Thanks for your reply, the problem has been solved. Thank you again for your dedication



---Original---
From: "Bartosz ***@***.***&gt;
Date: Thu, Dec 19, 2024 05:34 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [BartoszCichecki/LenovoLegionToolkit] Automatic power modeswitching (Discussion #1521)




 
Yes that is normal on some laptops. You can always configure an Action to switch to custom on startup.
